### PR TITLE
#164829157 Setup React Router Version 4 or Higher

### PR DESCRIPTION
**Pull request by koitoror**
#### What does this PR do?
It setups react router version 4 or higher

#### Description of Tasks to be completed?
Installs and configures the router and creates the  Header, Footer, Login and Home components and links to the Semantic-UI-CSS in order to create a dummy route for login and the "/" page to test the configuration.

#### How should this be manually tested?
- Clone the project: `git clone https://github.com/andela/ah-centauri-frontend`
- cd into the project folder
- checkout to branch `chore/164829157-react-router`
- Install dependencies: `npm install`
- Launch development server: `npm start`
- Opens the homepage on the browser you should see the navbar, a Home text and the footer.
- navigate to the login page on the top right navigation bar

#### Type of change
 - [x] chore(a non-breaking change that sets up react router).

#### What are the relevant Pivotal Tracker Stories?
[#164829157](https://www.pivotaltracker.com/story/show/164829157)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -30,7 +36,23 @@ if (token) {
 
 const App = () => (
   <Provider store={store}>
-    <div>Niaje wasee ndio hii hapa!</div>
+    <BrowserRouter>
+      <Container className="bg-gray">
+        <Header/>
+        <Switch>
+          {/* <Route exact path="/" component={HomePage}/> */}
```
Comment (bl4ck4ndbr0wn) [inline src/App.js]: remove commented out the code if not in use.

**Review by bl4ck4ndbr0wn — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (vino-jasuba) [inline src/App.js]: Please remove all instances of commented code

**Review by vino-jasuba — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (koitoror) [reply src/App.js]: Done.

**Review by vino-jasuba — APPROVED** [review]

**Review by bl4ck4ndbr0wn — APPROVED** [review]

File: src/components/layout/Header.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+          name="home"
+          active={activeItem === 'home'}
+          onClick={this.handleItemClick}
+        >
+          <Image
+            size="mini"
+            bordered
+            centered
+            circular
+            src="/images/logo.png"
```
Comment (Mirriam-Maina) [inline src/components/layout/Header.js]: @koitoror you will have to import the image from the images folder and pass it to the Image component as a prop `src={logo}`. The image is not accessible in this format.

**Review by Mirriam-Maina — CHANGES_REQUESTED** [review]
@koitoror you will have to import the image from the images folder and pass it to the Image component as a component src={logo}. The image is not accessible in this format.

File: src/components/layout/Footer.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+var style = {
```
Comment (Mirriam-Maina) [inline src/components/layout/Footer.js]: @koitoror Style this component with an external css file. You can assign the divs classNames

Comment (koitoror) [reply src/components/layout/Footer.js]: Done

File: src/components/layout/Header.js (hunk 2 above)
Comment (koitoror) [reply src/components/layout/Header.js]: Done

**Pull request merged**